src/visualization/src/window_delegate.h

```diff
@@ -0,0 +1,43 @@
+#ifndef WINDOW_DELEGATE_H_
+#define WINDOW_DELEGATE_H_
+
+#include "include/cef_app.h"
+#include "include/views/cef_window.h"
+#include "include/wrapper/cef_helpers.h"
+#include "include/base/cef_bind.h"
+#include "include/cef_app.h"
+#include "include/views/cef_browser_view.h"
+#include "include/views/cef_window.h"
+#include "include/wrapper/cef_closure_task.h"
+#include "include/wrapper/cef_helpers.h"
+
+class WindowDelegate : public CefWindowDelegate {
```

Comment:
I don't see any .cpp files using the WindowDelegate class. Is this header unused? If so, please remove it.

Reply:
Layer.cpp is using it

Reply:
Doesn't look like it to me. It's including this header, but the class doesn't appear to be used.